src/utils/helper.js

```diff
@@ -0,0 +1,14 @@
+export default function isEquivalent(a, b) {
```

Comment:
this seems to be doing the same thing as underscore's isEqual https://underscorejs.org/#isEqual
you can import this and use it instead